Directory.Build.props

```diff
@@ -84,7 +84,8 @@
     -->
     <Otherwise>
       <PropertyGroup>
-        <DefaultNetCoreTargetFramework>net8.0</DefaultNetCoreTargetFramework>
+        <DefaultNetCoreTargetFramework Condition="'$(IsVSCodeOnly)' == 'true'">net9.0</DefaultNetCoreTargetFramework>
+        <DefaultNetCoreTargetFramework Condition="'$(IsVSCodeOnly)' != 'true'">net8.0</DefaultNetCoreTargetFramework>
```

Comment:
From our Teams conversation the other day, I thought that this didn't actually work. My understanding of `Directory.Build.props` is that it is imported before the project file, and I would expect `IsVSCodeOnly` to always have no value. So, the projects would always be net8.0 and never net9.0. Am I missing something?

Reply:
Ugh, this still isn't up to date. I was pushing to my dev branch for integration testing and this PR isn't that branch

Reply:
Pushed correctly and actually verified in the browser this time

Reply:
Oh good! I was expecting different changes. Thanks!